content/blog/2018-10-02-outcomerate.md

```diff
@@ -0,0 +1,60 @@
+
+For a long time, it was common to find very different definitions of these rates. This made it hard to compare one survey to another. 
+
+## What Problem Does the Package Solve?
+
+The package calculates standard outcome rates in R in order to encourage transparent research, open methods, and scientific comparability. It does so using the [_American Association of Public Opinion Research_](https://www.aapor.org)'s (AAPOR) [definitions](https://www.aapor.org/Standards-Ethics/Standard-Definitions-(1).aspx) of the rates, which are the industry standard.
+
+Though no such R package existed previously, AAPOR does make available a [excel-based calculator](https://www.aapor.org/Education-Resources/For-Researchers/Poll-Survey-FAQ/Response-Rates-An-Overview.aspx). It is a wonderful free tool, but has some limitations. For one, the excel calculator is not designed to compute many rates by sub-domain; In my own work, I often try need rates in real time by day, by researcher/enumerator, and by geography. Doing each analysis in a separately excel workbook is impractical. Instead, [`outcomerate`](https://github.com/ropensci/outcomerate) makes it easy to achieve this with `dplyr`, `aggregate()`, or `data.table`. Second, Excel is  notorious for failing silently. It will give you a result even if the input contains errors or inconsistencies. This is dangerous because mistakes can creep into your work without your knowledge.
+
+The philosophy of the package is simple: let users flexibly calculate as many rates as possible with minimal effort, and fail fast. Unless specifically requested, the function only returns the rates that are defined based on the input, and provides soft warnings when it suspects users may be making errors. My hope is that this will encourage the frequent use of standard definitions, and facilitate their correct application.
+
+The package also implements weighted calculations of rates. Weighted figures further advance reproducible research because they (usually) estimate the outcome rate at the population level rather than for the sample at hand. This is useful information to share with the world, as it helps others plan for sample losses in future studies. Unless you plan to replicate the sampling design exactly, it is often more useful to know the (weighted) population rate.
+
+## Future Development
+
+In future releases of [`outcomerate`](https://github.com/ropensci/outcomerate), I may implement other sections from the AAPOR manual. Though arguably less popular than the “main” outcome rates, AAPOR has separate chapters for telephone surveys, online panels, and mixed-mode survey work. I chose not implement these initially, because some of the definitions differ despite using similar codes. It may also be worthwhile to add functions that make professional, publication-ready tables of the outcome rates. I am also considering functions that provide the standard error of the rates.
```

Comment:
Could you turn a couple of these into issues that others might consider working on?  Some may be code, but maybe others could be vignettes.
If yes, could you label for Hacktoberfest ? (See https://twitter.com/rOpenSci/status/1044708122265350145, https://hacktoberfest.digitalocean.com/)

Reply:
Excellent suggestion. I will prepare some issues and link them into the blog with an invitation to contribute.